src/args.jl

```diff
@@ -335,6 +335,12 @@ const _all_defaults = KW[
     _axis_defaults_byletter
 ]
 
+# to be able to reset font sizes to initial values
+const _initial_fontsizes = Dict(:titlefont  => deepcopy(_subplot_defaults[:titlefont]),
```

Comment:
yes, I can confirm this. you need a deepcopy (if you copy the entire type like I do). i'll chnage that.